pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/multithreading/xml/DoNotUseThreads.xml

```diff
@@ -29,32 +29,80 @@ extending threads is not allowed
 			]]>
         </code>
     </test-code>
-        <test-code>
+    <test-code>
         <description><![CDATA[
-implementing runnabel is not allowed
```

Comment:
I think, we should add a extra test case, that explicitly allows that runnable can be used (so keep this test case, but change the expected problems to 0) - referencing the github issue #1627 where is has been discussed. Without this test case, we don't have prove that this issue is fixed indeed (and stays fixed in the future).